exercises/error-handling/error_handling_test.go

```diff
@@ -11,7 +11,7 @@ import (
 // as defined by the expectations of this test suite.
 //
 // Also define a testVersion with a value that matches
-// the internal targetTestVersion here.
+// the targetTestVersion here.
```

Comment:
Yeah, I saw that too late. I made that change everywhere and was going to push it up but you were so quick to merge (and then I had to get on a plane). I'm now almost done with my 30+ hour trip. I'll fix the rest of these in the next day or so.